File: app/src/main/java/com/evaluation/model/result/picture/Picture.java
package com.evaluation.model.result.picture;

import com.google.gson.annotations.SerializedName;

/**
 * @author Vladyslav Havrylenko
 * @since 16.03.2020
 */
public class Picture {

    @SerializedName("large")
    private String mLarge;

    @SerializedName("medium")
    private String mMedium;

    @SerializedName("thumbnail")
    private String mThumbnail;

    public String getLarge() {
        return mLarge;
    }

    public String getMedium() {
        return mMedium;
    }

    public String getThumbnail() {
        return mThumbnail;
    }
}

File: app/src/main/java/com/evaluation/model/result/picture/Picture.kt
package com.evaluation.model.result.picture

import com.google.gson.annotations.SerializedName

/**
 * @author Vladyslav Havrylenko
 * @since 16.03.2020
 */
data class Picture(
    @SerializedName("large")
    val mLarge: String,

    @SerializedName("medium")
    val mMedium: String,

    @SerializedName("thumbnail")
    val mThumbnail: String
)